src/Table/Table.js

```diff
@@ -74,10 +74,8 @@ class Table extends React.Component<ProvidedProps & Props> {
     const divWrapper = classNames(classes.wrapper, classNameProp);
 
     return (
-      <div className={divWrapper}>
-        <ComponentProp className={classes.table} {...other}>
-          {children}
-        </ComponentProp>
+      <div className={divWrapper} {...other}>
```

Comment:
I still hate this. Maybe I should move the `classNameProp` variable and `{...other}` props back to `<ComponentProp>`?

Reply:
I hadn't appreciated the need for adding a wrapper. It explains why it was part of the demo, not the default behavior. We have other components that similarly require a suitably formatted container, such as [GridList](https://material-ui-next.com/demos/grid-list/#image-only-grid-list).

It seems right that <Table> should create a <table>.

Can we return to the beginning? What were you trying to achieve that the current Table doesn't already do?

Reply:
> Can we return to the beginning? What were you trying to achieve that the current Table doesn't already do?

The current table overflows by default which is kinda bad since this is a mobile optimized framework. So I requested a responsive table option in #9221 familar to the bootstrap responsive-table thing.

I made a PR for it and then you said, that tables should be always responsive since there is no reason not to have a responsive table at all, which is true.